packages/flutter/lib/src/material/text_field.dart

```diff
@@ -564,6 +564,15 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
     _editableTextKey.currentState?.requestKeyboard();
   }
 
+  void _handleSelectionChanged(TextSelection selection, SelectionChangedCause cause) {
+    // iOS cursor doesn't move via a selection handle. The scroll happens
+    // directly from new text selection changes.
+    if (Theme.of(context).platform == TargetPlatform.iOS
```

Comment:
Should this use a switch/case so the analyzer reminds us to consider what to do with newly added platforms here?